Cargo.toml

```diff
@@ -117,6 +117,8 @@ members = [
   "contracts/feature-tests/composability/transfer-role-features/meta",
   "contracts/feature-tests/composability/vault",
   "contracts/feature-tests/composability/vault/meta",
+  "contracts/feature-tests/contract-base-tester",
```

Comment:
Having an entire new contract just for this feature is a bit much.

You can use `use-module`, since it is also designed to test modules.

We should check that explicit ContractBase is allowed in both module and main contract.